cmd/lncli/commands.go

```diff
@@ -331,16 +332,26 @@ var sendCoinsCommand = cli.Command{
 				"scripts",
 		},
 		coinSelectionStrategyFlag,
+		cli.StringSliceFlag{
+			Name: "utxo",
```

Comment:
Ok will add.

Reply:
I think if we want to be precise here this flag should be named `outpoint` instead of `utxo` since a utxo usually contains more info like value, pkscript etc. including the outpoint.

Reply:
I think if we want to be precise here this flag should be named `outpoint` instead of `utxo` since a utxo usually contains more info like value, pkscript etc. including the outpoint.

Reply:
just that utxo is what was used for the channel funding one, so uniformity?